redbot/cogs/downloader/downloader.py

```diff
@@ -792,63 +808,66 @@ def __init__(self, bot: Red):
                         )
                     for page in pagify(msg):
                         await ctx.send(msg)
-                    return
```

Comment:
This `return` shouldn't be removed, it is supposed to fail preemptively - all the messages that get appended after this make no sense with this error, e.g.
```
Error: short sha1 `ab2c` is ambiguous. Possible candidates:
commit ab2cxyz - commit message
commit ab2cabc - other commit message
There were no cogs to check.
```
"There were no cogs to check." is not valid here.